nodes/nomos-node/src/api/handlers.rs

```diff
@@ -201,7 +201,7 @@ where
     post,
     path = "/da/add_blob",
     responses(
-        (status = 200, description = "Attestation for DA blob", body = Option<Attestation>),
+        (status = 200, description = "Blob to be published received"),
```

Comment:
Nit: We could readd the `body`, as we are actually responding with it, even though attestation in this case is `()` (unit type).

Reply:
> Nit: We could readd the `body`, as we are actually responding with it, even though attestation in this case is `()` (unit type).

I think we should remove the attestation though. Also we are just gonna remove this or change the endpoint so we actually do not need to return anything.

Reply:
Since we are still getting `DispersalResponse` from individual nodes, `Attestation` could be renamed to something similar. This probably could be done in a seperate PR.